yarn.lock

```diff
@@ -0,0 +1,8925 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please remove this file - these packages shouldn't be installed locally

Reply:
done